README.rst

```diff
@@ -29,13 +29,171 @@
+   managing requests.
+-  Managing patrons fees (overdue, subscriptions).
+-  Searching for documents, describing documents, linking authorities to
+   documents, importing metadata in MARC format, adding items to
+   documents.
+-  Setting serials patterns, receiving issues.
+-  Managing acquisition budgets, vendors information, orders and
+   invoices.
+
+How to install RERO ILS?
+------------------------
+
+The installation process is described in a `specific file <INSTALL.rst>`__.
+
+For a development environment you can check another `documentation <https://github.com/rero/developer-resources/blob/master/rero-instances/rero-ils/dev_installation.md>`__
+
+Check the `ecosystem <#the-rero-ils-ecosystem>`__ section.
+
+How to contribute to RERO ILS?
+------------------------------
+
+You can test the latest release on
+`ils.test.rero.ch <https://ils.test.rero.ch>`__.
+
+If you have question, you can ask the development team on
+`Gitter <https://gitter.im/rero/reroils>`__.
+
+You can also `open an issue <https://github.com/rero/rero-ils/issues/new>`__.
+
+To contribute to the code itself, please check the
+`contributing recommandations <https://github.com/rero/rero-ils/blob/dev/CONTRIBUTING.rst>`__
+
+The RERO ILS ecosystem
+----------------------
+
+Three Github repositories for RERO ILS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This `Github project <https://github.com/rero/rero-ils>`__ is the main projet
+for RERO ILS, basically it provides the backend. But to work on the frontend of
+the project, you will need also
+`rero-ils-ui <https://github.com/rero/rero-ils-ui>`__, which is based on
+`ng-core <https://github.com/rero/ng-core>`__.
+
+MEF
+^^^
+
+On top of that, RERO ILS make use of two other RERO projects. The first one is
+`MEF <https://github.com/rero/rero-mef>`__, a *Multilingual Entity File*, which
+provides authorities (or entities) to RERO ILS, in two languages (for now):
+French and German. This is used to link documents to controlled description of
+authors or subjects. MEF is aggregating several authority files, such as
+`IdRef <https://www.idref.fr/>`__,
+`BnF <https://www.bnf.fr/fr/donnees-autorite-bnf>`__,
+`GND <https://www.dnb.de/DE/Professionell/Standardisierung/GND/gnd_node.html>`__
+and `RERO <http://data.rero.ch/>`__. These authority files are then aligned
+through `VIAF <https://viaf.org>`__, thus providing authorities in several
+languages. At this point of the project, only physical persons have been added
+to MEF.
+
+As a result, to run RERO ILS, you need either to use our
+`public MEF server <https://mef.test.rero.ch>`__, or run your own.
+
+RERO EBOOKS
+^^^^^^^^^^^
+
+Some of our partners are customers of an ebooks commercial providers. We have
+set up a server that harvests the metadata of these ebooks, converts the
+metadata into our data model, and import them into our catalog, attributing
+each ebook to the library that subscribed to the platform. RERO ILS, then, is
+harvesting our ebooks server.
```

Comment:
```suggestion
harvests the ebooks server.
```

**NOTE:** It's not quite clear who does what. Is it the ebooks server that *imports them into the union catalog, assigning each ebook to the corresponding subscribing libraries*? Or is part of this job done by RERO ILS?

Reply:
You're right, I've reorganized this part, with the help of your corrections.